source/fundamentals/data-formats/uuids.txt

```diff
@@ -0,0 +1,569 @@
+   input_uuid = uuid4()
+   collection = client.testdb.get_collection('test', codec_options=std_opts)
+   collection.insert_one({'_id': 'baz', 'uuid': input_uuid})
+   assert collection.find_one({'uuid': Binary(input_uuid.bytes, 4)})['_id'] == 'baz'
+
+   # Retrieving this document using UuidRepresentation.JAVA_LEGACY returns a native UUID
+   # without modifying the UUID byte-order
+   java_opts = CodecOptions(uuid_representation=UuidRepresentation.JAVA_LEGACY)
+   java_collection = client.testdb.get_collection('test', codec_options=java_opts)
+   doc = java_collection.find_one({'_id': 'baz'})
+   assert doc['uuid'] == input_uuid
+
+   # Round-tripping the retrieved document silently changes the Binary bytes and subtype
+   java_collection.replace_one({'_id': 'baz'}, doc)
+   assert collection.find_one({'uuid': Binary(input_uuid.bytes, 3)}) is None
+   assert collection.find_one({'uuid': Binary(input_uuid.bytes, 4)}) is None
+   round_tripped_doc = collection.find_one({'_id': 'baz'})
+   assert round_tripped_doc['uuid'] == Binary(input_uuid.bytes, 3).as_uuid(UuidRepresentation.JAVA_LEGACY)
+
+In the preceding example, using the incorrect ``~bson.binary.UuidRepresentation``
+(``JAVA_LEGACY`` instead of ``STANDARD``) changes the
+``~bson.binary.Binary`` bytes and subtype as a side effect.
+This happens when any representation that
+manipulates byte order (``CSHARP_LEGACY`` or ``JAVA_LEGACY``) is incorrectly
+used to round-trip UUIDs written with the ``STANDARD`` representation.
+When the original document is written using the ``CSHARP_LEGACY``
+or ``JAVA_LEGACY`` representation and then round-tripped using the ``STANDARD``
+representation, only the ``~bson.binary.Binary`` subtype is changed.
+
+.. note::
+   
+   In {+driver-short+} 4.0+,
+   the ``STANDARD`` representation will decode binary subtype 3 fields as
+   ``~bson.binary.Binary`` objects of subtype 3 (instead of
+   ``uuid.UUID``), and each of the ``LEGACY_*`` representations will
+   decode binary subtype 4 fields to ``~bson.binary.Binary`` objects of
+   subtype 4 (instead of ``uuid.UUID``). This change resolves many of the preceding
+   issues.
+
+.. _configuring-uuid-representation:
+
+Configuring a UUID Representation
+---------------------------------
+
+You can work around the problems described above by configuring your
+applications with the appropriate ``~bson.binary.UuidRepresentation``.
+Configuring the representation modifies {+driver-short+}'s behavior while
+encoding ``uuid.UUID`` objects to BSON and decoding
+binary subtype 3 and 4 fields from BSON.
+
+You can set the UUID representation for your application in the following ways:
+
+- At the ``MongoClient`` level, by using the ``uuidRepresentation`` URI option:
+
+  .. code-block:: python
+
+     client = MongoClient("mongodb://a:27107/?uuidRepresentation=standard")
+
+  The following table describes the available values for the ``uuidRepresentation`` option:
+
+  .. list-table::
```

Comment:
There's still some weird rendering going on here (it's bolding some of the bullets like their titles). I think it's because these need to be unindented?

```suggestion
The following table describes the available values for the ``uuidRepresentation`` option:

.. list-table::
```